pyproject.toml

```diff
@@ -30,9 +30,8 @@ dependencies = [
     # scipy is currently bounded from above because scipy 1.11 removed the
     # `sym_pos` kwarg from `scipy.linalg.solve`, but pyscf has not yet been updated.
     "scipy>=1.5.2,<1.11",
-    "rustworkx>=0.12.0",
```

Comment:
I think bumping the rustworkx version to 0.13.0 was actually the right call, because we import it directly and therefore should list it as an explicit dependency rather than relying on it being a transitive dependency via qiskit (even though there is essentially no risk of that changing -- the most likely reason it might change, though, is it were ever renamed _again_).  0.13.0 is precisely what Qiskit 0.44.0 depends on, so that's the perfect version.  So I'd say `git revert e17fb62d6d1fccfbf3f560000d02c0e5d2c39319`.

Reply:
And yes, we had to bump because `qiskit 0.44` needs `rustworkx 0.13`